docs/content/pgo-client/common-tasks.md

```diff
@@ -292,10 +292,11 @@ pgo create cluster hacluster --cpu=4 --memory=16Gi
 #### Create a PostgreSQL Cluster with PostGIS
 
 To create a PostgreSQL cluster that uses the geospatial extension PostGIS, you
-can execute the following command:
+can execute the following command, updated with your desired image tag. In the
+example below, the cluster will use PostgreSQL 11.8 and PostGIS 2.5:
 
 ```shell
-pgo create cluster hagiscluster --ccp-image=crunchy-postgres-gis-ha
+pgo create cluster hagiscluster --ccp-image={{< param centosBase >}}-{{< param postgresVersion >}}-{{< param postgisVersion >}}-{{< param operatorVersion >}}
```

Comment:
Hmm, is this command correct? It seems to have removed the image name from it, and just has the tag.

```suggestion
pgo create cluster hagiscluster \
  --ccp-image=crunchy-postgres-gis-ha \
  --ccp-image-tag={{< param centosBase >}}-{{< param postgresVersion >}}-{{< param postgisVersion >}}-{{< param operatorVersion >}}
```

Reply:
Yes, not sure what happened there. Updated, thanks.